c3formatter.js

```diff
@@ -0,0 +1,51 @@
+function createDateLabels() {
```

Comment:
What's this file for? These 3 functions seem to be repeated in `app_modules/c3formatter.js`.